Dockerfile

```diff
@@ -14,6 +42,10 @@ RUN : \
 
 LABEL "org.opencontainers.image.version"="$IMAGE_VERSION"
 
+COPY --from=builder /opt/script /usr/bin/
+# Validate script binary
+RUN /usr/bin/script &>/dev/null
```

Comment:
Nit: this adds an extra layer, should we instead test the container with actual testing instead of using the Dockerfile to validate the binary?

Reply:
We certainly could, but I was worried that it would be "out of sight, out of mind" by anyone building the Dockerfile directly. What do folks think?

Reply:
I remember asking for this and hadn't considered the extra layer. I still kinda like the check here as a belt-and-suspenders method for ensuring things work. It might be nice to add a bit more to the comment explaining that we're doing this check on account of copying a dynamically linked binary.